collect/stressRelief.go

```diff
@@ -226,7 +226,7 @@ func (s *StressRelief) sigmoid(num, denom string) float64 {
 	r := s.ratio(num, denom)
 	// this is an S curve from 0 to 1, centered around 0.5 -- determined
 	// by messing around with a graphing calculator
-	stress := .395*math.Atan(6*(r-0.5)) + 0.5
+	stress := 0.400305589*math.Atan(6*(r-0.5)) + 0.5
```

Comment:
Any value in this scalar being defined in a constant somewhere?

Reply:
I didn't really think so -- it's just designed to be an S-shaped curve from 0 to 1, and symmetric around 0.5. There's no reason to ever change it. Does it need more comments?

Reply:
Nah I just wanted to make sure it wouldn't get used to scale something else similarly in the future.